aten/src/ATen/native/native_functions.yaml

```diff
@@ -2999,6 +2999,18 @@
   dispatch:
     CompositeExplicitAutograd: mkldnn_convolution_backward
 
+- func: mkldnn_convolution_transpose(Tensor self, Tensor weight, Tensor? bias, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: mkldnn_convolution_transpose
+
+- func: mkldnn_convolution_transpose_backward_input(int[] self_size, Tensor grad_output, Tensor weight, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups, bool bias_defined) -> Tensor
+
+- func: mkldnn_convolution_transpose_backward_weights(int[] weight_size, Tensor grad_output, Tensor self, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups, bool bias_defined) -> (Tensor, Tensor)
+
+- func: mkldnn_convolution_transpose_backward(Tensor self, Tensor grad_output, Tensor weight, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups, bool[3] output_mask) -> (Tensor, Tensor, Tensor)
+  dispatch:
+    CompositeExplicitAutograd: mkldnn_convolution_transpose_backward
```

Comment:
FYI I've been working to collapse the various convolution ops into a single general set of `convolution` + `convolution_backward` that handle routing to the correct backends.

Towards that effort, is it possible to demote the ops here to helper functions that are not exposed in `native_functions.yaml`?

Reply:
sure, i noticed your related PRs. Feel free to go ahead, i will rebase accordingly :)